modules/cnv-understanding-live-migration.adoc

```diff
@@ -5,16 +5,10 @@
 [id="cnv-understanding-live-migration_{context}"]
 = Understanding live migration
 
-Live migration is the process of moving a running virtual machine instance to 
-another node in the cluster without interruption to the virtual workload or 
```

Comment:
This might just be me being particular, but in general it would be better to edit text rather than block-copy the already existing text in a new PR. The reason being is that it means you can't use 'git blame' to easily figure out why something is written a certain way. This becomes important in subsequent releases when you need to follow the trail to a BZ or jira, and git blame is an easy way to see changes line-by-line and chase down the commit in which the text in question was introduced, which will have the ID. Replacing block text by copying existing content means git blame will cul-de-sac with this PR, which is unrelated.

Reply:
Gotcha, will just edit existing content in the future.